File: LLM-Translations/GPT-4/codenet/Python/C/s880494034.c

#include <stdio.h>
#include <math.h>

int main() {
    int N, i, j;
    scanf("%d", &N);
    
    int XS[N];
    for (i = 0; i < N; i++) {
        scanf("%d", &XS[i]);
    }

    int m = 999999999;
    for (i = 1; i <= 100; i++) {
        int c = 0;
        for (j = 0; j < N; j++) {
            c += pow(XS[j] - i, 2);
        }
        if (m > c) {
            m = c;
        }
    }

    printf("%d\n", m);

    return 0;
}

File: LLM-Translations/GPT-4/codenet/Python/Java/s880494034.java
import java.util.Scanner;

public class s880494034 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int N = scanner.nextInt();
        int[] XS = new int[N];
        for (int i = 0; i < N; i++) {
            XS[i] = scanner.nextInt();
        }

        int m = Integer.MAX_VALUE;
        for (int i = 1; i <= 100; i++) {
            int c = 0;
            for (int x : XS) {
                c += Math.pow(x - i, 2);
            }
            if (m > c) {
                m = c;
            }
        }

        System.out.println(m);
    }
}
// 
